<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Number Of Member in Team</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select Performance Type&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>Select Batch Year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B.Tech 1st Year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B.Tech 2nd Year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B.Tech 3rd Year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B.Tech 4th Year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M.Tech 1st Year</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M.Tech 2nd Year</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>450</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Marks Given By Judges</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Performance Number</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select Performance Type&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>Select Performance Type</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Group Music</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Group Dance </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Solo Music</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Solo Dance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Poetry</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Drama</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Standup Comedy</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
